handbook/engineering/security.md

```diff
@@ -0,0 +1,41 @@
+# Security
+
+We have [a bug bounty program](https://hackerone.com/sourcegraph).
+
+## Development
+
+Code reviews from senior engineers and peers are mandatory for any code to be merged (no direct
+pushes to `master`). Security-sensitive pull requests must undergo [review by the proper security
+code owner](code_reviews.md#security). Furthermore, we use Sourcegraph to provide
+critical context during code reviews (such as identifying dependencies of modified code).
+
+We use a number of static analysis tools to identify security risks in development, including the
+following:
+- language-specific linters
+- notifications and alerts for risky code patterns
+- code coverage tools to ensure unit test coverage
+- tools such as dependabot and GitHub security alerts to identify security vulnerabilities in our
+  code and in dependencies
+- enhanced code reviews via Sourcegraph.
+
+All development laptops have encrypted hard drives.
+
+## Ops
+
+Sourcegraph instances that host private code are deployed on-premises and therefore Sourcegraph
+employees have no access to customer data.
+
+On-premises Sourcegraph instances do not send any customer code to other servers. Additional
+customer private data such as email addresses is never sent (with the exception of emails collected
+explicitly for sales and support purposes).
+
+On sourcegraph.com, Sourcegraph uses role-based access controls to ensure only administrators have
+direct access to production compute resources and sensitive configuration data. All persistent
+volumes are encrypted.
```

Comment:
It is news to me that all persistent volumes are encrypted. Can you point me to the relevant Kubernetes manifests?